R/getDataDump.R

```diff
@@ -407,7 +407,5 @@ WHERE
   # Midlertidig fjerning av variabler i datadumpen
   tab %<>% dplyr::select(!dplyr::contains(c("Avdod", "AvdodDato")))
```

Comment:
Resultatet av ```dplyr::select()``` skal  her ikke tilordnes noe (_e.g._ tab). Standarden for en R-funksjon er at den returnerer sitt siste "statement". På denne linja skal det altså bare stå:
```dplyr::select(!dplyr::contains(c("Avdod", "AvdodDato")))```

Reply:
Da er dette endret